lib/info.js

```diff
@@ -0,0 +1,44 @@
+const axios = require('axios')
+const urljoin = require('url-join')
+const {getMetadata} = require('./get.js')
+const {customMarked} = require('./utils/marked.js')
+const {logger} = require('./utils/log-handler.js')
+const utils = require('../lib/utils/common')
+
+const getInfo = async (dhpkgid) => {
+  const { publisher, pkg } = utils.parseIdentifier(dhpkgid)
+
+  const metadata = await getMetadata(publisher, pkg)
+  const readme = await getReadme(urljoin(metadata.bitstore_url, 'README.md'))
+
+  const firstParagraphReadme = `# ABOUT\n${readme.substring(0, 200)}... *see more below*`
```

Comment:
Hm, now I'm thinking... what if there is no README at all?
We need one more check `if (readme) { ... }`